canonical-web-and-design/jaas-dashboard
# Add a report a bug link to the navigation

## Done
Add a report a bug link to the bottom of the navigation

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check the Report a bug link in the bottom of the navigation works

## Details
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1075
